src/components/TextInputWithPrefix/index.js

```diff
@@ -39,6 +39,10 @@ const TextInputWithPrefix = props => (
                     styles.textInputWithPrefix.textInput,
                     styles.noOutline,
                 ]}
+
+                // By default, align input to the left to override right alignment in RTL mode which is not yet supported in the App.
+                // eslint-disable-next-line react/jsx-props-no-multi-spaces
+                textAlign="left"
```

Comment:
Sorry, I might not have been clear enough... we should 

1. Create a new file
2. import the `TextInput` there
3. set the `textAlign="left"` passing all other props etc
4. import that _new file_ in all these different places instead of simply copying any pasting the same logic if every file that uses the `TextInput` directly

Does that work?

Reply:
yes, it's clear that's better the repeat ourselves. 

sure I will do it now.

Reply:
@parasharrajat  @marcaaron  it's done now. and work fine.